docs/architecture/deployment/emr.md

```diff
@@ -0,0 +1,67 @@
+
+## Create Amazon EMR cluster with Apache Livy
+
+:::caution Warning
+Livy does not work with Python version 3.8 and above.
+:::
+
+Navigate to Amazon EMR service and create a cluster. Under **Application bundle** select **Custom**.
+
+Choose appropriate applications to include in your installation. At a minimum, please make sure sure **Livy** , **Spark** and and **Hive** are included in the install.
+
+<img src={require('./img/livy.png').default} alt="EMR create cluster" width="75%" />
+
+### Configure connectivity between Livy and Prophecy
+
+To configure the necessary network settings for seamless integration, specific modifications to the security groups of your EMR cluster are required. Please follow the instructions outlined below:
+If you intend to utilize Prophecy as a SaaS (Software as a Service) solution, please note that the Prophecy public IP is `3.133.35.237`. Ensure that the Core security group's outbound rule allows connections to this IP address.
+
+1. Modify the **Primary Node** security group:
+   - Allow incoming connections to port `8998` from the Prophecy IP.
+   - This can be achieved by adding an inbound rule to the Master security group that permits incoming traffic on port `8998` from the Prophecy IP address.
+2. Modify the **Core Node** security group:
+   - Allow outgoing connections to the Prophecy public IP over HTTPS.
+   - To enable this, add an outbound rule to the Core security group that allows outgoing traffic over HTTPS protocol to the Prophecy public IP.
+
+By implementing these adjustments to your EMR cluster's security groups, you will establish the necessary network configuration for effective integration with Prophecy.
+
+## Create a Fabric to connect Prophecy to Livy
+
+Navigate to Prophecy's UI and click on **Create Fabric**. The Fabric will establish a connection with your EMR cluster and utilize it as the execution engine for your Pipelines.
+
+<img src={require('./img/livyfabric.png').default} alt="EMR create cluster" width="75%" />
+
+Choose **Livy** as your **Provider**.
+
+<img src={require('./img/provider.png').default} alt="EMR create cluster" width="75%" />
+
+Before proceeding, it is crucial to ensure that all the required settings are properly configured. If you are uncertain about the necessary configurations, we recommend reaching out to your cloud infrastructure team for additional guidance and assistance. They will be able to provide you with the specific information and support needed to ensure a successful setup.
+
+You may find your **Livy URL** under the **Applications** tab on your EMR Cluster page.
+
+<img src={require('./img/livyurl.png').default} alt="EMR create cluster" width="75%" />
+
+Copy/paste your Livy URL under _Spark Connection_ and choose the appropriate _Spark Version_ and _Scala Version_
+
+Configure appropriate **Prophecy Library** based on your Spark and Scala version.
```

Comment:
I'm concerned the user won't know what to paste into the library field. 

Is there a link to the library options?
Is there a suggestion to use latest, or a certain version?
When I search our docs, this doc also doesn't link to a list of libraries, so maybe I'm off the mark. https://docs.prophecy.io/concepts/fabrics/create-a-fabric/#databricks